webserver/templates/datasets/eval-job.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+
+{% block title %}Dataset "{{ dataset['name'] }}" by {{ author.musicbrainz_id }} - AcousticBrainz{% endblock %}
+
+{% block content %}
+  <div id="evaluation-results">
+
+    {% from 'datasets/macros.html' import print_header %}
+    {{ print_header(dataset, author, current_user, 'evaluation') }}
+
+    {% if job['status'] == 'pending' %}
+      <div id="status" class="alert alert-info">
+        This dataset has not been evaluated yet.
+      </div>
+
+    {% elif job['status'] == 'running' %}
+      <div id="status" class="alert alert-info">
+        This dataset is being evaluated right now. Results should be available soon.
+      </div>
+
+    {% elif job['status'] == 'failed' %}
+      <div id="status" class="alert alert-danger">
+        Evaluation of this dataset has failed!
```

Comment:
Yes, it's not user-friendly. But considering that Gaia doesn't have proper exceptions, there's not much else we can do there I think. String parsing seems very hacky and fragile.

I plan to create public pages with information about current evaluation jobs, that might be helpful. Perhaps we can show some additional functions for admins there.

Ticket for admin interface: http://tickets.musicbrainz.org/browse/AB-66. Feel free to add details.